### PR TITLE
fix(html): error handling for makeHtmlDriver method

### PR DESCRIPTION
**Pull request by desfero**
ISSUES CLOSED: #710

- [x] There exists an issue discussing the need for this PR
- [x] I added new tests for the issue I fixed or built
- [x] I ran `make test FOO` for the package FOO I'm modifying
- [x] I used `make commit` instead of `git commit`

### CODE REVIEW

File: html/src/makeHTMLDriver.ts [hunk 1]
```diff
@@ -42,9 +45,8 @@ export function makeHTMLDriver(
   function htmlDriver(vnode$: Stream<VNode>, name: string): HTMLSource {
     const html$ = vnode$.map(toHTML);
     html$.addListener({
-      next: effect || noop,
```
Comment (jvanbruegge) [inline html/src/makeHTMLDriver.ts]: Any reason why you removed the fallback?

Comment (desfero) [reply html/src/makeHTMLDriver.ts]: Is the fallback need as effect is required?

**Comment by staltz** [comment]
Hi @desfero :)
Thanks for submitting this PR. Before merging it could you notice that it's possible to add tests for this. Take a look at cycle/run tests or cycle/dom tests that stub `console.error` using `sinon` library.

File: html/test/index.ts [hunk 2]
```diff
@@ -237,4 +234,22 @@ describe('HTML Driver', function() {
...
+      return {
+        html: xs.of('invalid snabbdom' as any)
+      };
+    }
+
+    run(app, {
+      html: makeHTMLDriver(() => {}),
+    });
+
+    sinon.assert.calledOnce(console.error as any);
```
[the remark this answers is not in the document]
Comment (staltz) [reply html/test/index.ts]: Looks good to me! Thanks @desfero

**Pull request merged**